app/helpers/account_activations_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```

Comment:
I think you can delete this file if you do not use it

Reply:
I kept it out of hoarding, 'you never know when you might use it :laughing:'. Thanks, I'll delete them!